src/pages/news/2021-06-11-signalen-door-zijn-eerste-replicator-interview-erik-veerman.md

```diff
@@ -0,0 +1,51 @@
+
+KMS (Klacht en Meldingensysteem) bij de Afvalstoffendienst en Melddesk.
+
+2. Waarom heb je besloten om te upgraden naar Signalen?
+
+We hebben onderzocht of de huidige oplossingen nog voldoen aan de wensen van de organisatie. De oude oplossingen voldeden niet meer aan de eisen, vooral op het gebied van integratie met andere applicaties. Ook hadden we de wens om beter te kunnen communiceren met de reporters (melders). Wat niet echt een optie was met de oplossingen die we toen hadden.
+
+3. Waarom is een replicatie beter dan vanaf nul beginnen in je eigen omgeving?
+
+Het grootste voordeel van replicatie van Signalen is dat **veel van de gewenste functionaliteit al is gerealiseerd en bewezen in Amsterdam**. Als je vanaf nul begint, is er veel meer onzekerheid of en hoe de oplossing zal eindigen. En replicatie heeft waarschijnlijk een financieel voordeel, hoewel ik niet zeker weet of dit waar is.
+
+4. Wat waren de belangrijkste uitdagingen?
+
+De twee grootste uitdagingen waren:
+     a. Vind een balans tussen ons uitgebreide eisen- en wensenlijstje en de nog te ontwikkelen of aan te passen functionaliteit.
+     b. Zoveel mogelijk onderdelen en functionaliteit van SIA geschikt maken voor gebruik in 's-Hertogenbosch (en andere gemeenten) en toch blijven fuseren tot één Signalen masterbranche zodat alle Signalen deelnemers elkaar kunnen ontwikkelen. Vooral in het begin was er spanning met Amsterdam omdat Signalen (SIA) hun creatie is en aanpassingen aan Signalen door of voor anderen dan als moeilijk ervaren kunnen worden.
+    
+5. Toekomstperspectieven voor 's-Hertogenbosch
+
+Wat Signalen betreft hebben we nog een uitgebreide lijst met gewenste functionaliteiten. Deze willen we het liefst samen met andere gemeenten ontwikkelen.
+
+6. Voordelen van het gebruik van open source-technologieën in een replicatie als deze
+
+Goede vraag! Ik weet het niet zeker. Maar ik denk dat het makkelijker is om met open source technologie een duurzame omgeving/gemeenschap van softwareleveranciers en andere gemeenten te creëren.
+
+7. Wat zou je veranderen als je helemaal opnieuw zou beginnen?
+
+In het begin hadden we niet echt een idee hoe Signalen werkte, wat de mogelijkheden waren en wat de achterliggende gedachten van bepaalde keuzes waren. Hierdoor was het moeilijk om te bepalen wat Signalen van onze eisen en wensen te bieden had. Dit veroorzaakte ook enkele problemen bij het testen van de applicatie. Bij de aanschaf van een softwareoplossing krijgt u ondersteuning van een soort implementatieadviseur. Als deze rol aan het begin van het traject bij Signalen beschikbaar was geweest, was het traject een stuk eenvoudiger geweest. Dus als ik het opnieuw zou kunnen doen, zou ik zo snel mogelijk iemand aan boord halen met (proces)kennis van Signalen.
+
+8. Tips die je een volgende replicator zou geven
+
+Zie hierboven en probeer informatie en documentatie te krijgen die door andere gemeenten wordt gebruikt.
```

Comment:
probeer informatie en documentatie te krijgen die door andere gemeenten wordt gebruikt. 
Suggest "probeer informatie en documentatie in te winnen bij andere gemeenten die Signalen gebruiken"